### DISCUSSION TITLE
Please return the commas in the prompt

### DISCUSSION THREAD

**Discussion by R2AZ3** [zanllp/infinite-image-browsing#710] [General]
Hello, I am Korean, and I use this extension very well. I also have introduced this extension to the Korean AI community. 
However, at some point, the prompts of 'Structured Data' started to be displayed without commas (,). But I really need those commas. 
I usually drag and copy parts of the prompts from existing images, but now they are copied without commas, making them difficult to use. Of course, I can copy with commas in 'Source Text', but it makes the prompts visually difficult to see. 
Before, when there were commas, I thought it was a fantastic feature because it was easy to see and copy and use. So I hope you bring back the commas. If you think it is a visual distraction, set the color of commas to white. Then, they will be invisible, but you can copy them by dragging.
![image](https://github.com/user-attachments/assets/fef0d863-8e6e-46e2-ad5c-12bf4f0c8f2f)

**Comment by zanllp** [comment]
Thank you for helping to promote IIB

This change started from commit 1e2925df0111de56327da16c0fade88d6252431a. It does seem to have some impact. I can't directly revert it, but perhaps I can provide an option for you to choose.

**Reply by R2AZ3** [reply]
I would really appreciate it if you could make it optional in the settings. That would be awesome. Of course this extension is already awesome!

**Comment by zanllp** [comment]
<img width="682" alt="image" src="https://github.com/user-attachments/assets/04cb2302-8807-46fb-a9ea-2a6d6450ac98">

**Reply by R2AZ3** [reply]
Thank you so much for making this option. I just tried it and it's awesome!!
